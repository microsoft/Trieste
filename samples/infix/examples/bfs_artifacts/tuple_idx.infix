foo = ((1,),);bar = 1 . 1 . (1,);