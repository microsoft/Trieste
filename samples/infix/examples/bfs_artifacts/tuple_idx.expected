//! --enable-tuples --tuples-require-parens parse_only
//! --enable-tuples --tuples-require-parens --use-parser-tuples parse_only
//! --enable-tuples parse_only
(top
  {}
  (infix-calculation
    {
      bar = infix-assign
      foo = infix-assign}
    (infix-assign
      (infix-ident 3:foo)
      (infix-expression
        (infix-tuple
          (infix-expression
            (infix-tuple
              (infix-expression
                (infix-int 1:1)))))))
    (infix-assign
      (infix-ident 3:bar)
      (infix-expression
        (infix-tupleidx
          (infix-expression
            (infix-tupleidx
              (infix-expression
                (infix-int 1:1))
              (infix-expression
                (infix-int 1:1))))
          (infix-expression
            (infix-tuple
              (infix-expression
                (infix-int 1:1)))))))))
